pom.xml

```diff
@@ -40,7 +40,7 @@
   <modules>
     <module>apiary-metastore-events</module>
     <module>apiary-metastore-metrics</module>
-    <module>hive-metastore-events</module>
+    <module>hive-event-listeners</module>
```

Comment:
The parent module under here that this is effectively pointing at should be renamed from `hive-metastore-events-parent` to `hive-event-listeners-parent` to be consistent (and the child modules need their parent updated accordingly). And this is then a non-backwards compatible change so the version number of this project should go to `7.0.0`.